Prace_domowe/Praca_domowa3/Grupa4/Adrian_Stańdo/pd03.ipynb

```diff
@@ -0,0 +1,859 @@
+{
```

Comment:
<p>Fajna dyskusja, to prawda, że wszystko zależy od "celu biznesowego"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>